### DISCUSSION TITLE
am i a moron or dose android mobile login not work?

### DISCUSSION THREAD

**Discussion by Uncommon-34** [balzack/databag#110] [Q&A] [answered]
so I'm trying to login using the android mobile app, I already made an account in the browser and now trying to login. I have tried the URL to the server then my account pass word didn't work, the server URL and the accounts mobile token didn't work, then also just my username and password didn't work and finally the account name and token didn't work. I also tried making a new account using the app they worked just fine and was messaging everything just fine. so just wondering if I'm being a moron and the solution Is simple I just don't see it or is there an issue here?

**Comment by Sathvik-Rao** [comment, marked as answer]
If for example your server running databag is on URL - sample.com
Your user account username is - user1
and password is - pass

In the mobile app login as:
Username(username/URL): user1/sample.com
Password: pass

**Reply by Uncommon-34** [reply]
ahhhh that's what it means by username/URL (i didn't think it would  be that as with create account URL had its own box) I wasn't doing that lol thank you!

**Reply by balzack** [reply]
good point, I see how that is confusing (slash can mean one or the other).

I previously had it as username@host, but thought that was confusing people with email, so I changed the hint (although @ still works as a separator)

**Reply by Uncommon-34** [reply]
i think @ is a better separator but in my ideal it would be a separate input so a third bot instead kind of like how it is in the create account page

**Reply by balzack** [reply]
I agree, the separate input is a good idea. I'm working on a large refactor, and will include that change. Thanks!

**Comment by balzack** [comment]
Thanks for helping with the response. I was traveling and just getting to this now. Please let me know if this is still an issue.

**Comment by brando-bang** [comment]
Hello! I recently updated my iOS app and am having trouble with the new login screen (with server and username separated) - I’m getting an Operation Failed popup whenever I try

just wanted to ask a few questions to try to narrow it down:
Is mobile back to requiring https?
Does the url allow IP or does it require domain names
Should I include the port or will it default to 7000 

some more info: I don’t reverse proxy my home server but do access it remotely over tailscale. I don’t use certificates but if https is required I can add them 

e:

Disregard, I just saw the https requirement is back due to fdroid requirements. I get how it could be annoying to change http/https requirements across different releases for different appstores so i will just setup https on my end.

**Comment by balzack** [comment]
I agree with you that the fdroid requirement should not cascade to an iOS requirement. I will look into this, sorry it's not working for you at the moment.

**Reply by brando-bang** [reply]
np and thanks. For now this was the motivation i needed to finally setup tailscale serve on some of my homelab services haha
